src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -1317,6 +1318,11 @@ struct SmartTarget
         {
             uint32 seat;
         } vehicle;
```

Comment:
what is this union member used for?

Reply:
Used by SMART_TARGET_VEHICLE_ACCESSORY:
`
if (Unit* target = me->GetVehicleKit()->GetPassenger(e.target.vehicle.seat))
`

Reply:
how many scripts in current 3.3.5 tdb use this target?

Reply:
Nothing at all

Reply:
remove it then (replace it in the enum with `_UNUSED`)